src/Components/Search/NewSearch/NewSearchBar.tsx

```diff
@@ -0,0 +1,35 @@
+import { AutocompleteInput, Box } from "@artsy/palette"
+import { ClassI18n } from "System/i18n/ClassI18n"
+import { Media } from "Utils/Responsive"
+
+export const NewSearchBar = () => {
+  const handleChange = ({ target: { value } }) => {
+    console.log("Seach input value ", value)
+  }
+
+  const renderAutosuggestComponent = (t, { xs }) => {
+    return (
+      <AutocompleteInput
+        placeholder={xs ? t`navbar.searchArtsy` : t`navbar.searchBy`}
+        spellCheck={false}
+        options={[]}
+        onChange={handleChange}
+      />
+    )
+  }
+
+  return (
+    <Box flex={1}>
```

Comment:
i think it's better to use `Flex` component

Reply:
With Flex we need more props. Box is better in my opinion.